### ISSUE TITLE
Prompt pasting does not work with non-standard prompt

### ISSUE THREAD

**Issue by dpsanders** [KristofferC/OhMyREPL.jl#54]
I changed my Julia 0.5 prompt to `julia v0.5>`. 
Pasting `julia v0.5> x = 3` at a prompt now breaks:

```
julia v0.5> julia v0
syntax: extra token "v0" after end of expression

julia v0.5> .5> x = 3
syntax: "0.5" is not a valid function argument name
```

**Comment by KristofferC** [comment]
Can you please try #55. You should be able to paste custom prompts as well as "julia> " prompts.

**Comment by dpsanders** [comment]
Seems to be working great, thanks!

**Comment by KristofferC** [comment]
Closed in #55
